features/gezag-minderjarige/4b.1-ouder-of-partner-overleden-of-onbevoegd-tot-gezag.feature

```diff
@@ -39,10 +39,15 @@ Functionaliteit: 4b.1 - Ouder of partner overleden of onbevoegd tot gezag
   • Meerderjarig verklaring: ophalen meerderjarig verklaring uit de aantekening in het Gezagsregister
 
   Gebruikte velden:
+    - Geboortedatum persoon -> 01.03.10
+    - Burgerservicenummer ouder 1 -> 02.01.20
+    - Burgerservicenummer ouder 1 -> 03.01.20
+    - Burgerservicenummer huwelijk/geregistreerd partnerschap -> 05.01.20
+    - Start datum huwelijk/geregistreerd partnerschap -> 05.06.10
```

Comment:
in deze  feature (/ deze stap in de gezagsmodule) worden geen huwelijk of partnerschap velden gebruikt

Reply:
Discussie in https://github.com/BRP-API/brp-api-gezag/pull/254/files#r1867612640.

Deze openhouden ter referentie voor toekomstige wijzigingen.